src/rpc/server.cpp

```diff
@@ -217,7 +217,11 @@ UniValue help(const UniValue& params, bool fHelp)
             "mining --------> Returns help for staking/cpid/beacon related commands\n"
             "developer -----> Returns help for developer commands\n"
             "network -------> Returns help for network related commands\n"
-            "voting --------> Returns help for voting related commands\n";
+            "voting --------> Returns help for voting related commands\n"
+	    "\n"
+	    "The Gridcoin Foundation invites to donate to address\n"
+	    "bc3NA8e8E3EoTL1qhRmeprbjWcmuoZ26A2\n"
```

Comment:
Change the wording to...

You can support the development of Gridcoin by donating GRC to the Gridcoin Foundation at this address:

Reply:
Did you change it and re-push?

Reply:
It just arrived. I put the line break just before "Gridcoin Foundation" - did not want to separate these words. Googled the hash - looks fine to me.